oap-server/server-starter/src/main/resources/ui-initialized-templates/k8s_service/k8s-service-pod.json

```
@@ -1636,8 +1636,36 @@
       "layer": "K8S_SERVICE",
       "entity": "ServiceInstance",
       "name": "K8S-Service-Pods",
-      "id": "89a35b46-449d-4ef0-8297-2109f69086eb",
-      "isRoot": false
+      "isDefault": true,
+      "expressions": [
+        "avg(kubernetes_service_instance_write_package_size)/60",
+        "avg(kubernetes_service_instance_read_package_size)/60",
+        "avg(kubernetes_service_instance_http_call_cpm)",
+        "avg(kubernetes_service_instance_http_call_duration/kubernetes_service_instance_http_call_cpm/1000000)",
+        "avg(kubernetes_service_instance_http_call_success_count / kubernetes_service_instance_http_call_cpm*100)"
+      ],
+      "expressionsConfig": [
+        {
+          "unit": "bytes",
+          "label": "Write Per Seconds"
+        },
+        {
+          "unit": "bytes",
+          "label": "Read Per Seconds"
+        },
```

```suggestion
        {
          "unit": "bytes",
          "label": "Write Per Second"
        },
        {
          "unit": "bytes",
          "label": "Read Per Second"
        },
```

No `s`. And TCP load and HTTP load are using different style of unit, could you align them?